contentcuration/contentcuration/frontend/shared/data/serverSync.js

```diff
@@ -343,8 +386,11 @@ async function handleChanges(changes) {
 
   // If we detect  changes were written to the changes table
   // then we'll trigger sync
-  if (syncableChanges.length || newChangeTableEntries) {
-    debouncedSyncChanges();
+
+  if (newChangeTableEntries.length) {
+    if (!syncActive) {
+      WebsocketSendChanges();
+    }
```

Comment:
One issue I see with this is that if new changes come in while the `syncActive` is `true`, then those would have to wait until a new change event in order to trigger syncing through the websocket. @rtibbles if we figure out how to properly include the `client_rev`, could this go back to just passing `newChangetableEntries.map(...)`?